src/DisplayWindow.cpp

```diff
@@ -75,8 +75,9 @@ bool DisplayWindow::resizeWindow()
 	if (!m_bResizeWindow)
 		return false;
 	m_drawer._destroyData();
-	if (!_resizeWindow())
-		_start();
+	if (!m_bFullscreen)
+		if (!_resizeWindow())
```

Comment:
Won't this mean that if ```m_bFullscreen == true```, _resizeWindow() will return false, and thus _start() will be called, shouldn't it just do nothing?